gateway_test.go

```diff
@@ -750,6 +750,50 @@ func TestQuota(t *testing.T) {
 	}
 }
 
+func TestDistributedQuotaSingleNode(t *testing.T) {
+	spec := createSpecTest(t, nonExpiringDefNoWhiteList)
+	session := createQuotaSession()
+	keyId := testKey(t, "key")
+	spec.SessionManager.UpdateSession(keyId, session, 60)
+	defer spec.SessionManager.ResetQuota(keyId, session)
+
+	recorder := httptest.NewRecorder()
+	param := make(url.Values)
+	req, err := http.NewRequest("GET", param.Encode(), nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	req.Header.Add("authorization", keyId)
+
+	config.DQSetMaster = true
+	config.UseDistributedQuotaCounter = true
+	config.DistributedQuotaFlushIntervalInMS = 100
```

Comment:
these are not reset, so they modify the global config for the rest of the tests. You should either set them globally, or set back to the original values in this test via a `defer`.